### DISCUSSION TITLE
what is the meaning of `explicitlyDeleted=false` in LDBC?

### DISCUSSION THREAD

**Discussion by chakpongchung** [ldbc/ldbc_snb_docs#229] [Q&A]
Hi LDBC community!

I looked at the LDBC benchmark doc but it seems like it is missing some details. I want to understand how entries are/should be ingested to measure performance.

Question details are in the link below. Let me know if you prefer the question to be posted here instead.

https://stackoverflow.com/questions/75038145/what-is-the-meaning-of-explicitlydeleted-false-in-ldbc?noredirect=1#comment132427222_75038145

**Comment by szarnyasg** [comment]
@chakpongchung thanks, good question. I wrote an answer on SO. I will turn this discussion into an issue because this should be explained in the documentation.
